tests/src/main/java/com/hazelcast/stabilizer/tests/map/helpers/IntegerStringEntryListener.java

```diff
@@ -67,31 +67,11 @@ public void mapCleared(MapEvent mapEvent) {
     }
 
     private void delay() {
-        if (maxDelayMs != 0) {
+        if (minDelayMs >= 0 && maxDelayMs > 0) {
```

Comment:
Nope? This check should just prevent negative values from being used. `minDelayMs` is totally fine with `0` and `maxDelayMs` was not allowed to be zero before.